packages/storybook/.eslintrc.json

```
@@ -0,0 +1,4 @@
+{
+  "extends": ["plugin:@fluentui/eslint-plugin/react"],
```

The `node` config might be more appropriate here since this package is used for dev-time stuff. (`node` might not be the best name for the config.)
```suggestion
  "extends": ["plugin:@fluentui/eslint-plugin/node"],
```

---

since so far the package only contains shared addon code (which is React). i think we should keep `react`. perhaps we will need to also extend `node` later